yb-voyager/cmd/root.go

```diff
@@ -227,7 +227,7 @@ func registerCommonGlobalFlags(cmd *cobra.Command) {
 
 	registerExportDirFlag(cmd)
 
-	cmd.PersistentFlags().StringVarP(&config.LogLevel, "log-level", "l", "info",
+	cmd.PersistentFlags().StringVarP(&config.LogLevel, "log-level", "l", "debug",
```

Comment:
Are you setting the default log-level as debug? @makalaaneesh

Reply:
Just to run tests to ensure nothing is breaking. Will revert this before merging.

Reply:
Wait, sorry that doesn't make sense. The code will run irrespective of the log level 🤔 
I've already verified the log output manually, so should be fine.